### ISSUE TITLE
`fp/no-unused-expression` with `'use strict'`

### ISSUE THREAD

**Issue by nfantone** [jfmengels/eslint-plugin-fp#56]
![image](https://user-images.githubusercontent.com/476069/143563468-15918abd-9fd2-4912-93d6-413d17f9e511.png)

Consider making `allowUseStrict` options `true` by default (or making that part of `plugin:fp/recommended`). `'use strict'` should never be flagged as an "unused expression".

**Comment by nfantone** [comment]
Same reasoning applies for `module.exports` on CJS.

```sh
Unallowed reassignment. You may want to activate the `commonjs` option for this ruleeslint(fp/no-mutation)
```

IMHO, this should be either baked into the plugin logic or be turned on by default.
